listen.js

```diff
@@ -0,0 +1,6 @@
+const app = require("./app.js");
+
+app.listen(8080, (err) => {
```

Comment:
Don't think you need this app.listen() - only useful if you're using Insomnia/some other manual endpoint testing. Supertest is acting as the client for you.

Reply:
can I delete the whole listen file?